doc/classes/EditorSyntaxHighlighter.xml

```diff
@@ -13,13 +13,14 @@
 		<method name="_get_name" qualifiers="virtual const">
 			<return type="String" />
 			<description>
-				Virtual method which can be overridden to return the syntax highlighter name.
+				Virtual method which can be overridden to return the syntax highlighter name. This is a name that is displayed in the editor when selecting between syntax highlighters, e.g. [code]Markdown[/code].
```

Comment:
I don't think there's anywhere else where _"syntax highlighter name"_ would matter as an term worth remembering. You may as well merge the two sentences.
There's a bit of ambiguity because `e.g.` may refer to "name that is displayed", but also "syntax highlighters". Not sure how to solve that, but maybe:
```suggestion
				Virtual method which can be overridden to return the name that is displayed in the editor when selecting between syntax highlighters. For example:  [code]"Markdown"[/code].
```